extensions/amp-skimlinks/0.1/affiliate-link-resolver.js

```diff
@@ -107,7 +107,7 @@ export class AffiliateLinkResolver {
    *
    * @public
    * @param {Array<HTMLElement>} anchorList
-   * @return {*}
+   * @return {!../../../src/service/link-rewriter/link-rewriter.TwoStepsResponse}
```

Comment:
This notation is horrible but we can't do anything about it. Moving files around must be a pain when you need to update all the js docs...